python/test/test_package.py

```diff
@@ -14,7 +20,17 @@ def get_spec_version(cls, filename):
 
     def test_version_consensus(self):
         from ga4gh.dos import __version__
-        cwd = os.path.dirname(os.path.realpath(__file__))
-        spec_dir = os.path.join(cwd, "../../openapi")
-        for f in os.listdir(spec_dir):
-            assert __version__ == self.get_spec_version(os.path.join(spec_dir, f))
+        for f in os.listdir(self.spec_dir):
+            assert __version__ == self.get_spec_version(os.path.join(self.spec_dir, f))
+
+    def test_schema_validity(self):
+        """Check to make sure that the Swagger schema specification is
+        valid by running it through the online validator. If the schema
+        is valid, the online validator will return 200 OK and an empty
+        JSON response."""
+        path = os.path.join(self.spec_dir, 'data_object_service.swagger.yaml')
```

Comment:
Thank you, this surface is currently embarrassingly untested and these tests will improve the experience of managing schema changes (since you can run `nostests test/test_package`).

I think it's worth seeing if you can use [swagger-spec-validator](https://pypi.org/project/swagger-spec-validator/) instead of hitting a service. We're implicitly depending on it through bravado:

https://github.com/Yelp/bravado-core/blob/master/setup.py#L19